sources/playground/utils/timer.sql

```diff
@@ -0,0 +1,6 @@
+@echo c, "#include <utils/timer.c>\n"; 
+
+declare function create_timer() object<timer> not null;
```

Comment:
This is wrong you have to say

declare function create_timer() CREATE object<timer> not null;

The caller will own the reference.  You do this on all create methods.

Reply:
If you don't do this you leak a timer on every call.

Reply:
You might want to figure out some way in the playground to say that Lua isn't supported for some samples.

Reply:
Since the only issue is the CREATE thing I'll just accept the pull and fix that for you.

Reply:
pulled and fixed